src/lobby/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as core from '../core'
+import * as dns from '../dns'
+import { dataRoot } from '../data-root'
+
+import { USERNAME_BLOCKLIST } from './blocklist'
+
+
+/**
+ * Create a user account.
+ */
+export const createAccount = async (
+  userProps: { email: string, username: string },
+  apiEndpoint?: string,
+  apiDid?: string
+): Promise<any> => {
+  apiDid = apiDid || await core.apiDid()
+  apiEndpoint = apiEndpoint || core.apiEndpoint()
+
+  const jwt = await core.ucan({
+    audience: apiDid,
+    issuer: await core.did(),
+  })
+
+  return fetch(`${apiEndpoint}/user`, {
```

Comment:
Something I've been thinking about is that we're really prioritizing browsers right now. (which makes sense, we're building a browser-native sdk), but there's nothing stopping a Node application from using our sdk in the future. All of our crypto stuff is only available in browsers so we're not Node-compatible by any means right now, but eventually we'll probably want to switch this over to `axios` or something similar. 

Anyways nothing to do here atm, I'm just rambling :stuck_out_tongue: